pkg/releaser/git.go

```diff
@@ -157,6 +163,47 @@ func (r *Releaser) submitPR(request *source.ReleaseRequest) (string, error) {
 	return pr.GetHTMLURL(), nil
 }
 
+func (r *Releaser) closeExistingPR(request *source.ReleaseRequest, client *github.Client) error {
+	queryString := fmt.Sprintf("is:pr is:open author:%s repo:%s/%s release new version %s",
+		request.PluginReleaseActor,
+		r.UpstreamKrewIndexRepoOwner,
+		r.UpstreamKrewIndexRepo,
+		request.PluginName,
+	)
+	closeComment := fmt.Sprintf("Closing this PR as it's outdated\n\n/close")
```

Comment:
i am not sure if `/close` will work as I don't think the bot have access to close the PR in krew-index repo.

Reply:
Also I am wondering if we should close the PR after the new PR is opened. this way we can add the new PR link while closing the old PR's?

Reply:
We can reference the old PR.

Also prow (which is used in the krew-index repo) understands the /close command when they're issued by a Repo member or the owner of the PR, so this should work fine. 

Anyway I can open and close a test PR in the repo (if this is ok to @ahmetb) just to check if this works fine :)